client/src/pages/UserPage.tsx

```diff
@@ -0,0 +1,276 @@
+                      data: anxiety,
+                      borderColor: 'rgba(6, 24, 187, 0.17)',
+                      backgroundColor: 'rgba(41, 58, 249, 0.53)',
+                      fill: '-1',
+                      borderWidth: 1.2,
+                      tension: 0.4,
+                      order: 2,
+                    },
+                    {
+                      label: 'Physical Activity',
+                      data: physical,
+                      borderColor: 'rgb(161, 1, 1)',
+                      backgroundColor: 'rgba(255, 4, 4, 0.63)',
+                      fill: 'origin',
+                      tension: 0.4,
+                      borderWidth: 1,
+                      order: 3,
+                    },
+                    {
+                      label: 'Dream Quality',
+                      data: dreamActivity,
+                      borderColor: 'rgb(255, 213, 0)',
+                      backgroundColor: 'rgba(255, 183, 0, 0.96)',
+                      fill: 'origin',
+                      borderWidth: 1.2,
+                      tension: 0.4,
+                      order: 3,
+                    },
+                  ],
+                }}
+                options={{
+                  responsive: true,
+
+                  scales: {
+                    x: {
+                      grid: {
+                        display: false,
+                      },
+                    },
+                    y: {
+                      title: {
+                        display: true,
+                        text: 'Progress',
+                      },
+                    },
+                  },
+
+                  plugins: {
+                    filler: {
+                      propagate: true,
+                    },
+                    legend: {
+                      position: 'bottom',
+                    },
+                    tooltip: {
+                      backgroundColor: 'rgb(5, 3, 29)',
+                      borderWidth: 1,
+                      cornerRadius: 6,
+                      boxHeight: 5,
+                      padding: 4,
+                      caretSize: 4,
+                      caretPadding: 2,
+                      titleFont: {
+                        size: 10,
+                        family: 'Helvetica Neue',
+                        weight: 'bold',
+                      },
+                      displayColors: false,
+                      yAlign: 'bottom',
+                      xAlign: 'left',
+                      bodySpacing: 2,
+                      titleSpacing: 2,
+                      enabled: true,
+                      mode: 'index',
+                      axis: 'y',
+                      intersect: false,
+                      position: 'nearest',
+                      bodyFont: {
+                        size: 8,
+                      },
+                      callbacks: {
+                        title: function (tooltipItems) {
```

Comment:
Just use arrow functions here. This makes it obvious you just copied the code from somewhere.